app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,33 @@
+#chess_board, .black, .white {
+	display: block; 
+	padding: 0px;
+	margin: 0px; 
+}
+
+#chess_board { 
+	width: 640px; 
+	height: 640px;
+	border: 4px solid black; 
+	padding: 0px;
+	margin: 0px; 
+} 
+
+.black, .white {
+	width: 80px; 
+	height:80px; 
+	float: left; 
+	padding: 0px;
+	margin: 0px;
+} 
+
+.black {
+	background-color: black; 
+}
+
+.white {
+	background-color: #FFD966;
+} 
+
+table {
+	border-collapse:collapse;
+}
```

Comment:
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline